Packages/com.unity.inputsystem/Documentation~/input-actions-editor-window-reference.md

```diff
@@ -0,0 +1,32 @@
+# Input Actions Editor reference
+
+The **Input Actions Editor** is an Editor window that displays action maps, actions, bindings, and their properties.
+
+![The Input Actions editor, displaying the default actions](./Images/ActionsEditorCallout.png)
```

Comment:
To make the caption render properly, I think you need to insert a line break like so:

```
![The Input Actions editor, displaying the default actions](./Images/ActionsEditorCallout.png)</br>*The Input Actions editor, displaying the default actions*
```